contrib/oc-environment.sh

```diff
@@ -31,6 +31,9 @@ export BRIDGE_K8S_MODE_OFF_CLUSTER_THANOS
 BRIDGE_K8S_MODE_OFF_CLUSTER_ALERTMANAGER=$(oc -n openshift-config-managed get configmap monitoring-shared-config -o jsonpath='{.data.alertmanagerPublicURL}')
 export BRIDGE_K8S_MODE_OFF_CLUSTER_ALERTMANAGER
 
+BRIDGE_K8S_MODE_OFF_CLUSTER_GITOPS=$(oc -n openshift-pipelines-app-delivery get route cluster -o jsonpath='{.spec.host}' 2> /dev/null)
```

Comment:
We should fix the script to prepend `https://` here since you only have the hostname.

Reply:
`https://` is prepended in the script